src/client/components/Map/Map.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component, createRef } from 'react';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
+import { Icon } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import logo from '../../assets/images/treasure.png';
+
+const treasureIcon = new Icon({
+  iconUrl: logo,
+  iconSize: [71, 71],
+});
+
+class Mymap extends Component {
```

Comment:
Should these be hardcoded?

Reply:
No, it should not but back then i did not had that icon which suppose to go there, so i picked another icon and fixed the size